### ISSUE TITLE
Lyric started by [A-G] and without Chord Over throw exception [Chords Over Words Parser]

### ISSUE THREAD

**Issue by acneidert** [martijnversluis/ChordSheetJS#821]
This part throw exception because lyric start with "A" and don't have chords over. Throws not only in initial verse, but in middle too
```
title: Let it be
key: C

And when the broken-hearted people
 Am      F      F6
Living in the world agree
 C         G      F Em/F F6 C
There will be an answer, let it be
```

**Comment by davehell** [comment]
The same problem is for the lines started by [1-7] in Chords Over Words Parser
`1. line of the first paragraph`

**Comment by martijnversluis** [comment]
Hey @davehell. Thanks for reaching out. I'm currently working on a refactor where I also stumbled upon this issue. I'll link them so you're keep in the loop 👍

**Comment by mholta** [comment]
Thank you for this fantastic library @martijnversluis!🔥 Is there any way to contribute in solving this issue?

**Comment by martijnversluis** [comment]
Hey @mholta. Thanks for reaching out.

There are ways! The issue is in the [PegJS grammar for chords over words](https://github.com/martijnversluis/ChordSheetJS/blob/master/src/parser/chords_over_words_grammar.pegjs). I tries to distinguish lyrics from chords, but when a word is also a valid chord parsing goes south.

So if you have some PegJS experience, or if you know somebody who does, that could be a great help in solving the issue!

**Comment by martijnversluis** [comment]
This issue has been solved! 🎉  If you update to `8.2.0` these chord sheets are correctly parsed 👍
